news-app-diptimehta/src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Navbar.css";
+export const Navbar = () => {
+  return (
+    <div>
+      <ul className="ul-navbar">
+        <li className="li-navbar">
```

Comment:
Try to make an array of objects and then loop through it through map

Example:
`const routes = [
{
    name: 'Home'
    route: '/'
    className: 'right'
}
]`